### ISSUE TITLE
z1: default example broken

### ISSUE THREAD

**Issue by OlegHahm** [RIOT-OS/RIOT#6713]
After flashing the default example to my z1 node, the only output I get is:
```
2017-03-07 13:32:55,993 - INFO # RIOT MSP430 hardware initialization complete.
```
and nothing else happens. Shell is also not responding. Flashing shell test, for instance, works.

Investigating further details.

**Comment by OlegHahm** [comment]
After bisecting I conclude the SPI rework seems to have introduced this bug.

**Comment by smlng** [comment]
didn't @A-Paul worked with the Z1 nodes, maybe he or @dnahm can look into this?

**Comment by haukepetersen** [comment]
fixed by #7355

**Comment by aabadie** [comment]
Since #7355 is merged, I close this one.
